### ISSUE TITLE
"Unable to properly filter input" on blocks at a later date.

### ISSUE THREAD

**Issue by kenjicncr** [subquery/subql#823]
I'm getting an `Unable to properly filter input` after a certain number of blocks. 

The filter input is fine about 5-6 days ago. But anything after that throws this error.

Here's the link to the repo

https://github.com/kenjicncr/blockplace-subgraph

**Comment by kenjicncr** [comment]
Did a little digging and the tx seems to be returning undefined. Here's what the input data looks like. Looks like i'm getting legacy transaction input?

```
{"legacy":{"nonce":73,"gasPrice":1000000000,"gasLimit":996942,"action":{"call":"0x11a7927e08d2f5c99659481d3f0e01c5e553eb06"},"value":0,"input":"0x69af27db00000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000002c40000000000000000000000000000000000000000000000000000000000000007500000000000000000000000000000000000000000000000000000000000005910000000000000000000000000000000000000000000000000000000000000007000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000591000000000000000000000000000000000000000000000000000000000000000b000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000005920000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000005920000000000000000000000000000000000000000000000000000000000000005000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000005920000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000005920000000000000000000000000000000000000000000000000000000000000007000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000005920000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000005920000000000000000000000000000000000000000000000000000000000000009000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000592000000000000000000000000000000000000000000000000000000000000000a000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000592000000000000000000000000000000000000000000000000000000000000000c000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000592000000000000000000000000000000000000000000000000000000000000000d000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000592000000000000000000000000000000000000000000000000000000000000000e000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000592000000000000000000000000000000000000000000000000000000000000000f000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000005930000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000005930000000000000000000000000000000000000000000000000000000000000005000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000005930000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000005930000000000000000000000000000000000000000000000000000000000000007000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000593000000000000000000000000000000000000000000000000000000000000000c000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000593000000000000000000000000000000000000000000000000000000000000000d000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000593000000000000000000000000000000000000000000000000000000000000000e000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000593000000000000000000000000000000000000000000000000000000000000000f000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000005940000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000005940000000000000000000000000000000000000000000000000000000000000005000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000005940000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000005940000000000000000000000000000000000000000000000000000000000000007000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000005940000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000005940000000000000000000000000000000000000000000000000000000000000009000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000594000000000000000000000000000000000000000000000000000000000000000a000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000594000000000000000000000000000000000000000000000000000000000000000b000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000594000000000000000000000000000000000000000000000000000000000000000c000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000594000000000000000000000000000000000000000000000000000000000000000d000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000594000000000000000000000000000000000000000000000000000000000000000e000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000594000000000000000000000000000000000000000000000000000000000000000f000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000005950000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000005950000000000000000000000000000000000000000000000000000000000000005000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000005950000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000005950000000000000000000000000000000000000000000000000000000000000009000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000595000000000000000000000000000000000000000000000000000000000000000d000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006120000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006120000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006120000000000000000000000000000000000000000000000000000000000000003000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006120000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006120000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006120000000000000000000000000000000000000000000000000000000000000007000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006120000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000612000000000000000000000000000000000000000000000000000000000000000a000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000612000000000000000000000000000000000000000000000000000000000000000b000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000612000000000000000000000000000000000000000000000000000000000000000d000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006130000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006130000000000000000000000000000000000000000000000000000000000000009000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000613000000000000000000000000000000000000000000000000000000000000000c000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000613000000000000000000000000000000000000000000000000000000000000000d000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000613000000000000000000000000000000000000000000000000000000000000000e000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000613000000000000000000000000000000000000000000000000000000000000000f000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006140000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006140000000000000000000000000000000000000000000000000000000000000003000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006140000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006140000000000000000000000000000000000000000000000000000000000000007000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006140000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006140000000000000000000000000000000000000000000000000000000000000009000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000614000000000000000000000000000000000000000000000000000000000000000b000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000614000000000000000000000000000000000000000000000000000000000000000c000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000614000000000000000000000000000000000000000000000000000000000000000d000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000614000000000000000000000000000000000000000000000000000000000000000e000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000614000000000000000000000000000000000000000000000000000000000000000f000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006150000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006150000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006150000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006150000000000000000000000000000000000000000000000000000000000000005000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006150000000000000000000000000000000000000000000000000000000000000009000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000615000000000000000000000000000000000000000000000000000000000000000c000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000615000000000000000000000000000000000000000000000000000000000000000d000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006920000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006920000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006920000000000000000000000000000000000000000000000000000000000000007000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000692000000000000000000000000000000000000000000000000000000000000000a000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000692000000000000000000000000000000000000000000000000000000000000000b000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000692000000000000000000000000000000000000000000000000000000000000000f000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006930000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006930000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006930000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006930000000000000000000000000000000000000000000000000000000000000003000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006930000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006930000000000000000000000000000000000000000000000000000000000000005000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006930000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006930000000000000000000000000000000000000000000000000000000000000007000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006930000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006940000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006940000000000000000000000000000000000000000000000000000000000000003000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006940000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006940000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006940000000000000000000000000000000000000000000000000000000000000009000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000694000000000000000000000000000000000000000000000000000000000000000b000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000694000000000000000000000000000000000000000000000000000000000000000d000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000694000000000000000000000000000000000000000000000000000000000000000e000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006950000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006950000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006950000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006950000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006950000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006950000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000006950000000000000000000000000000000000000000000000000000000000000009000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000695000000000000000000000000000000000000000000000000000000000000000d000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000007120000000000000000000000000000000000000000000000000000000000000003000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000007130000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000007130000000000000000000000000000000000000000000000000000000000000005000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000007130000000000000000000000000000000000000000000000000000000000000007000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000007130000000000000000000000000000000000000000000000000000000000000009000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000713000000000000000000000000000000000000000000000000000000000000000a000000000000000000000000000000000000000000000000000000000000ebea0000000000000000000000000000000000000000000000000000000000000713000000000000000000000000000000000000000000000000000000000000000b000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000007140000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000007140000000000000000000000000000000000000000000000000000000000000003000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000007140000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000007140000000000000000000000000000000000000000000000000000000000000005000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000007140000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000007140000000000000000000000000000000000000000000000000000000000000007000000000000000000000000000000000000000000000000000000000000ebea00000000000000000000000000000000000000000000000000000000000007150000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000ebea000000000000000000000000000000000000000000000000000000000000000a4368617269747920233400000000000000000000000000000000000000000000","signature":{"v":2605,"r":"0x67bb752bd558d03a8a4cf3479922fdf06329ca3bcb1ad67a4d932cc44d468ad3","s":"0x37283c2ffee5c2551539d7547934ba661b9743e4ff9e178400b1b038c75091ae"}}} 
```

**Comment by stwiname** [comment]
Fixed in `@subql/contract-processors@0.5.2`
